templates/package.json

```diff
@@ -45,6 +45,24 @@
         "tab/js/m365",
         "tab/ts/m365"
     ],
+    "templatesV3": [
```

Comment:
seem the sync-up script need to update. 
these tabs also need to sync the sdk version
you can ask @hund030 for more info

Reply:
Good points! I think it's ok to update the sync-up script later and ensure the release process works.

Reply:
Create a follow up task here: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16017883